### PR TITLE
curvefs metaserver mds code

### PR DESCRIPTION
**Pull request by cw123**
### What problem does this PR solve?

Problem Summary: curvefs metaserver mds frame
 
### What is changed and how it works?

What's Changed: curvefs metaserver mds frame

How it Works: curvefs metaserver mds frame

Side effects(Breaking backward compatibility? Performance regression?):  no

### Check List

- [x] Relevant documentation/comments is changed or added
- [x] I acknowledge that all my contributions will be made under the project's license

### CODE REVIEW

**Comment by cw123** [comment]
recheck

File: curvefs/src/mds/fs.h [hunk 1]
```diff
@@ -0,0 +1,51 @@
...
+namespace curvefs {
+namespace mds {
+class MdsFsInfo {
+ public:
+    MdsFsInfo(uint32_t fsId, std::string fsName, uint64_t rootInodeId,
+             uint64_t capacity, uint64_t blockSize, common::Volume volume);
+    void ConvertToProto(FsInfo *file);
+    void CopyFromProto(const FsInfo file);
+
+ public:
```
Comment (aspirer) [inline curvefs/src/mds/fs.h]: should be private?

File: curvefs/src/mds/fs_manager.cpp [hunk 2]
```diff
@@ -0,0 +1,146 @@
...
+ * Created Date: 2021-05-19
+ * Author: chenwei
+ */
+#include <glog/logging.h>
+#include "curvefs/src/mds/fs_manager.h"
+namespace curvefs {
+namespace mds {
+FSStatusCode FsManager::CreateFs(std::string fsName, uint64_t blockSize,
+                curvefs::common::Volume volume, FsInfo* fsInfo) {
+    // 1、查询fs
```
Comment (aspirer) [inline curvefs/src/mds/fs_manager.cpp:28]: should add english comments, change all below.

File: buildfs.sh [hunk 3]
```diff
@@ -0,0 +1,35 @@
...
+    exit
+fi
+bazel build curvefs/proto:mds_cc_proto
+if [ $? -ne 0 ]
+then
+    echo "build mds proto failed"
+    exit
+fi
+
+bazel build curvefs/src/metaserver:main  --copt -DHAVE_ZLIB=1 --compilation_mode=dbg -s --define=with_glog=true --define=libunwind=true --copt -DGFLAGS_NS=google --copt -Wno-error=format-security --copt -DUSE_BTHREAD_MUTEX --copt -DCURVEVERSION=${curve_version} --linkopt -L/usr/local/lib
```
Comment (aspirer) [inline buildfs.sh:23]: --compilation_mode=dbg，
does this just use for dev stage? release package will be debug mode or not?

File: curvefs/src/mds/fs.h (hunk 1 above)
Comment (cw123) [reply curvefs/src/mds/fs.h]: done

File: buildfs.sh (hunk 3 above)
Comment (cw123) [reply buildfs.sh]: done

File: curvefs/src/mds/fs_manager.cpp (hunk 2 above)
Comment (cw123) [reply curvefs/src/mds/fs_manager.cpp]: done

**Comment by cw123** [comment]
recheck

**Comment by cw123** [comment]
new pr is here  #403

**Pull request closed without merging**